libraries/AP_HAL_ESP32/targets/esp-idf/project_description.json

```diff
@@ -0,0 +1,20 @@
+{
+    "project_name":       "ardupilot",
+    "project_path":       "/home/buzz/ardupilot/libraries/AP_HAL_ESP32/targets/esp-idf/",
+    "build_dir":          "/home/buzz/ardupilot/build/esp32buzz/esp-idf_build",
+    "config_file":        "/home/buzz/ardupilot/libraries/AP_HAL_ESP32/targets/esp-idf/sdkconfig",
+    "config_defaults":    "",
+    "app_elf":            "/home/buzz/ardupilot/build/esp32buzz/esp-idf_build/ardupilot.elf",
+    "app_bin":            "/home/buzz/ardupilot/build/esp32buzz/esp-idf_build/ardupilot.bin",
+    "git_revision":       "v4.2.4-209-g527a23d63f",
+    "target":             "esp32",
+    "phy_data_partition": "",
+    "monitor_baud" :      "115200",
+    "monitor_toolprefix": "xtensa-esp32-elf-",
+    "config_environment" : {
+        "COMPONENT_KCONFIGS" : "/home/buzz/ardupilot/modules/esp_idf/components/efuse/Kconfig;/home/buzz/ardupilot/modules/esp_idf/components/esp32/Kconfig;/home/buzz/ardupilot/modules/esp_idf/components/esp_common/Kconfig;/home/buzz/ardupilot/modules/esp_idf/components/log/Kconfig;/home/buzz/ardupilot/modules/esp_idf/components/spi_flash/Kconfig",
+        "COMPONENT_KCONFIGS_PROJBUILD" : "/home/buzz/ardupilot/modules/esp_idf/components/bootloader/Kconfig.projbuild;/home/buzz/ardupilot/modules/esp_idf/components/esptool_py/Kconfig.projbuild;/home/buzz/ardupilot/modules/esp_idf/components/partition_table/Kconfig.projbuild"
+    },
+    "build_components" : [ "bootloader", "bootloader_support", "efuse", "esp32", "esp_common", "esp_rom", "esptool_py", "log", "main", "micro-ecc", "partition_table", "soc", "spi_flash", "xtensa", "" ],
+    "build_component_paths" : [ "/home/buzz/ardupilot/modules/esp_idf/components/bootloader", "/home/buzz/ardupilot/modules/esp_idf/components/bootloader_support", "/home/buzz/ardupilot/modules/esp_idf/components/efuse", "/home/buzz/ardupilot/modules/esp_idf/components/esp32", "/home/buzz/ardupilot/modules/esp_idf/components/esp_common", "/home/buzz/ardupilot/modules/esp_idf/components/esp_rom", "/home/buzz/ardupilot/modules/esp_idf/components/esptool_py", "/home/buzz/ardupilot/modules/esp_idf/components/log", "/home/buzz/ardupilot/modules/esp_idf/components/bootloader/subproject/main", "/home/buzz/ardupilot/modules/esp_idf/components/bootloader/subproject/components/micro-ecc", "/home/buzz/ardupilot/modules/esp_idf/components/partition_table", "/home/buzz/ardupilot/modules/esp_idf/components/soc", "/home/buzz/ardupilot/modules/esp_idf/components/spi_flash", "/home/buzz/ardupilot/modules/esp_idf/components/xtensa", "" ]
```

Comment:
only users called buzz can build? :-)

Reply:
I've removed that file/commit, it was a template for something still a bit wip.